source/_integrations/fritz.markdown

```diff
@@ -66,6 +66,15 @@ If you have a dynamic IP address, most likely it will change.
 | ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------- |
 | `entity_id`            | no       | Only act on a specific  router                                                                                 |
 
+## Integration Options
+
+It is possible to change some behaviors through the integration options.
+These can be changed at **AVM FRITZ!Box Tools** -> **Configure** on the Integrations page.
+
+- **Consider home**: Number of seconds that must elapse before considering a disconnected device "not at home"
+
+**Note**: if you don't want to automatically track new detected device, disable the integration system option `Enable new added entities`
```

Comment:
With home-assistant/core#50791 this note is not needed, since all device_trackers were disabled by default

Reply:
We should say the other way around:

**if you want to automatically track new detected device, enable the integration system option `Enable new added entities`**

Reply:
sounds good 👍

Reply:
I think latest version clarify everything. What you think ?